agent-bundle/scripts/windows/make.ps1

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Splunk, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+<#
+.PARAMETER Target
+    Build target to run (bundle)
+#>
+param(
+    [Parameter(Mandatory=$true, Position=1)][string]$Target,
+    [Parameter(Mandatory=$false, ValueFromRemainingArguments=$true)]$Remaining
+)
+
+Set-PSDebug -Trace 1
+$BUNDLE_NAME = "agent-bundle_windows_amd64.zip"
```

Comment:
super nit, can we settle on hyphens or underscores or is one being used for a separator pattern?

Reply:
I'm just using `<dir name>_<os>_<arch>`, where `<dir name>` has been `agent-bundle`, and `_<os>_<arch>` is consistent with the binaries, e.g. `otelcol_windows_amd64.exe`.